api/v1alpha1/contour_types.go

```diff
@@ -83,11 +87,36 @@ type NamespaceSpec struct {
 	RemoveOnDeletion bool `json:"removeOnDeletion,omitempty"`
 }
 
+const (
+	// Available indicates that the contour is running and available.
+	ContourAvailableConditionType = "Available"
+)
+
 // ContourStatus defines the observed state of Contour.
 type ContourStatus struct {
-	// availableReplicas is the number of observed available Contour replicas
-	// according to the deployment.
-	AvailableReplicas int32 `json:"availableReplicas"`
+	// availableContours is the number of observed available replicas
+	// according to the Contour deployment. The deployment and its pods
+	// will reside in the namespace specified by spec.namespace.name of
+	// the contour.
+	AvailableContours int32 `json:"AvailableContours"`
```

Comment:
I like the rename here! 👍 

We should think about future stats like "requestedContours" or something to contrast how many are healthy vs requested.

Reply:
Requested Contour replicas would be in the spec, right?